apps/events/models.py

```diff
@@ -15,7 +15,7 @@ class Event(models.Model):
     schedule = models.DateTimeField()
     venue = models.CharField(max_length=50)
     thumbnail = models.ImageField(
-        upload_to='events/images', null=True, blank=True)
+        upload_to='<event>.get_thumbnail_url', null=True, blank=True)
```

Comment:
This isn't what I meant. Keep `upload_to=event_thumbnail_path` here.

I meant, if you want to get the URL of the thumbnail image anywhere in views or templates, you can just use the `get_thumbnail_url` property. For ex, if `event` is an instance of `Event` table, you can do `event.get_thumbnail_url` and it will return you the URL of thumbnail of that event.